crates/p2p_protocol/protocol/transport_dm.capnp

```diff
@@ -0,0 +1,100 @@
+# For Holochain, the app-level connections use DirectMessage for node2node
+# communications between app developer code.
+# At the transport-level, once a connection is established between two nodes,
+# We use DirectMessage within lib3h to set up multiplexing channels,
+# and negotiate relay services, etc.
+
+@0xdd5a14cf7a6734dc;
+
+struct Message {
```

Comment:
Suggest renaming to `TransportMessage` as `Message` sounds too generic and also collisions with `Message` in `p2p.capnp`.

Reply:
I think I'm going to rename this whole file to `multiplex.capnp` then, this struct name can be `MultiplexMessage` :+1:

Reply:
👍